### ISSUE TITLE
Import/export themes

### ISSUE THREAD

**Issue by skipmeister123** [felixse/FluentTerminal#17]
I would like to be able to import and export color themes I've created.

**Comment by skipmeister123** [comment]
It would be really fantastic if there was a way to import the themes from here https://iterm2colorschemes.com/

**Comment by felixse** [comment]
Yes, should be pretty easy to write a parser for those. You want to have a try on this?

**Comment by skipmeister123** [comment]
I don't think I'd be much help here.  I don't know C# :\

**Comment by felixse** [comment]
No problem, just thought so because of your PR 😃
